### DISCUSSION TITLE
props sent to plausible not showing up in goals

### DISCUSSION THREAD

**Discussion by yuejiang-code** [plausible/analytics#570] [Anything Else]
Hi,

I created a goal, viewReport, in my self hosted plausible. I then call `plausible('viewReport', { props: { hello: 'world' }})` but I don't see my data being sent in the API call, nor do I see it on plausible. Can you help?

Thanks
Yue

**Comment by metmarkosaric** [comment]
thanks @yuejiang-code! Custom props are not yet available on self-hosting, only on the cloud. 

You can find the self-hosting releases here: https://github.com/plausible/analytics/releases 

And you can learn about our self-hosting release schedule here: https://plausible.io/blog/building-open-source
